docker/owlbot/java/README.md

```diff
@@ -19,6 +19,38 @@ docker run --rm -v $(pwd):/workspace --user "$(id -u):$(id -g)" gcr.io/repo-auto
 
 ### Building the image
 
+#### Local Docker
+
+From the root of the synthtool repository, run:
+
+```bash
+suztomo@suztomo:~/synthtool$ docker build -f docker/owlbot/java/Dockerfile .
```

Comment:
nit: maybe omit the username and the hostname (also throughout the doc)

Reply:
+1

Reply:
Removed the user name references in this document.